.eslintignore

```diff
@@ -1,6 +1,10 @@
 # Dependency directories
 node_modules/
 
+# Styling
+*.scss

```

Comment:
**eslint** is not compatible with the **scss** files and cannot parse them; hence, the rule. The **master** branch **eslint** implementation do not throw any errors because currently it is not functioning correctly (e.g. if I put multiple semi-colons and more tabs then required in **src/index.js** file at some line, it will still not signal any issues).

Reply:
Introduced **stylelint* for styles linting.